software/25.Flutter/SETUP.md

```diff
@@ -0,0 +1,77 @@
+# Setup
+
+## Android Studio
+
+In this workshop, you'll have to install:
+- [Android Studio](https://developer.android.com/studio): Android apps IDE
+- Android SDK: Android development tools
+- [VSCode](https://code.visualstudio.com/): Visual Studio Code
+- [Chromium](https://www.chromium.org/chromium-projects/): Chromium browser
+- [Flutter](https://docs.flutter.dev/get-started/install/linux): A cross-platform framework for building mobile apps
+- [Dart](https://dart.dev/get-dart): A modern programming language for building apps
+
+To install Android SDK:
+- Open `Android Studio`
+- Go to `File -> Settings -> Appearance & Behavior -> System Settings -> Android SDK`
+- Click on `SDK Tools` tab
+- Select `Android SDK Command-line Tools`
+- Click on `Apply -> OK -> Finish` buttons
```

Comment:
For Android Studio i think it could be interesting to show how we can launch the project directly in the Idea

Reply:
You mean IDE ?